process/logsevents/fungibleESDTProcessor.go

```diff
@@ -82,33 +82,29 @@ func (fep *fungibleESDTProcessor) processEventOnSenderShard(event coreData.Event
 	})
 }
 
-func (fep *fungibleESDTProcessor) processEventDestination(args *argsProcessEvent, senderShardID uint32, selfShardID uint32) (string, string) {
+func (fep *fungibleESDTProcessor) processEventDestination(
+	args *argsProcessEvent,
+	selfShardID uint32,
+) (string, string, string, uint32) {
```

Comment:
could have used a struct instead of returning 4 values

Reply:
this function will be refactored in the next PR